internal/kafka/test/common/kafka_pollers.go

```diff
@@ -87,11 +91,16 @@ func WaitForKafkaToReachStatus(ctx context.Context, db *db.ConnectionFactory, cl
 			}
 		}).
 		OnRetry(func(attempt int, maxRetries int) (done bool, err error) {
-			kafka, _, err = client.DefaultApi.GetKafkaById(ctx, kafkaId)
+			k, resp, err := client.DefaultApi.GetKafkaById(ctx, kafkaId)
```

Comment:
why `k`, couldn't `kafka` be used?

Reply:
I had to replace it with `k` because now I had to use `:=` to create the `resp` variable.
By doing that, however, `kafka` become a new variable in the inner scope and was hiding the outer scope ones.

Reply:
Thanks.